thirdeye/thirdeye-pinot/src/main/resources/assets/css/thirdeye.css

```diff
@@ -91,6 +97,23 @@ body {
     margin-bottom: 0px;
     border: 0px;
 }
+
+.navbar-inverse .navbar-brand {
+  color: white;
+}
+
+.navbar-brand:before {
```

Comment:
Agreed that an actual `img` tag is preferred for actual images. But since this is the linkedin logo (next to "ThirdEye"), I feel that the element it is attached to "thirdeye" already conveys meaning. Do you see it differently?